.circleci/config.yml

```diff
@@ -37,6 +37,22 @@ jobs:
                           make check CLANG_FORMAT=clang-format-8
                           ./upload-stats.sh
                   fi
+      - run:
+          name: Build fuzz targets and deploy to Fuzzit
+          command: |
+                  make tests/fuzz-ppm tests/fuzz-jpg tests/fuzz-header tests/fuzz-text tests/fuzz-dstr CLANG=clang-8 CLANG_FORMAT=clang-format-8
+                  # TODO: CHANGE THIS TO CIRCLE SECRET
+                  export FUZZIT_KEY=055d9d3ba41a36fe7e33fee3bf411b2102703c60e16b1756a7222ab529f16dbf8c687a35313f0b1666d8687e891e5c83
+                  export ORG_ID=sjk2xLZ7DGKxBaTDwfZW
+                  wget -O fuzzit https://bin.fuzzit.dev/fuzzit-1.0
```

Comment:
I'm not entirely keen on doing this via wget in the the .circleci/config.yml. I'd rather put this into the Docker image and then use it from there.

Reply:
Fine by me. Though it's pretty standard I can add an md5 check if you like. I'll add another upodate soon as I want to update the version of the CLI to 1.1. (If it will be inside the docker it would be harder to update versions)

Reply:
Ok. I've added in the variables for my Fuzzit login to the circleci config. You should be able to use FUZZIT_KEY and FUZZIT_ORG.

Reply:
I've added you to the PDFGen project on fuzzit. can you go to the [settings](http://localhost:3000/admin/sjk2xLZ7DGKxBaTDwfZW/settings) page and update the FUZZIT_KEY please.

Reply:
I've done this now, and re-run the CI job.

Reply:
I updated the cli to 1.1 But I don't seed to be able to access $FUZZIT_KEY (looks like it is empty). Is this the correct name?

https://circleci.com/gh/fuzzitdev/PDFGen/8

Reply:
Isn't link is to your own circleci instance, rather than my one? Your one won't have the keys set up. I'm not entirely clear how that works. I've double checked, and I do have FUZZIT_KEY and FUZZIT_ORG set.

Reply:
@AndreRenaud I think it's ready now to be merge. The reason it didn't work is keys are not exposed to pull-request. I change the to a create only key so it should not be a problem that it is exposed.

Reply:
Ok. I guess the correct thing to do really is to put the keys back in as variables, and then limit this to only run on the master branch, not pull requests. For now I'll just leave it and we'll see how it goes.